docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -1,58 +1,67 @@
+Using the Docker with a Cloud Bucket as Remote Datasource
+=========================================================
 
 Introduction
 ------------
 The Lightly Docker can be used with the Lightly Platform to do
 the following workloads in one single run:
 
-- stream your files directly from your AWS S3 bucket to your local machine without
+- stream your files directly from your cloud bucket to your local machine without
   needing to sync or download them
 - embed all images or video frames
 - sample a subset, e.g. using coreset
 - compute the metadata of the images
 - create a dataset in the Lightly Platform from the selected subset
 
-Support for streaming from Google Cloud Storage and Azure Blob Storage is
-planned. If you need any of these, write us so that we prioritize implementing it.
+Lightly supports the following cloud storage solutions:
+
+- `AWS Secure Storage Solution (S3) <https://aws.amazon.com/s3/>`_
+- `Google Cloud Storage (GCS) <https://cloud.google.com/storage>`_
+- `Azure Blob Storage (Azure) <https://azure.microsoft.com/services/storage/blobs/>`_
 
 
 Advantages
 ----------
 
 - You can run the whole Lightly workflow with one single command.
 - You can process videos directly without needing to extract frames.
-- Your data is streamed from your S3 bucket on the go instead of being first downloaded
-  to your local disk and then read from there. Thus you save both a lot of time and
+- Your data is streamed from your bucket on the go instead of being first downloaded
+  to your local disk and then read from there. You save both a lot of time and
   disk space and can process much larger datasets.
-- You can :ref:`ref-docker-with-datasource-datapool`
+- You can :ref:`process new data in your bucket using a datapool <ref-docker-with-datasource-datapool>`
   to continuously update and improve your selected dataset
-  everytime new data comes into your S3 bucket.
-- Your images and videos are never saved anywhere but in your S3 bucket,
+  everytime new data comes into your bucket.
+- Your images and videos are never saved anywhere but in your bucket,
   maintaining your privacy and security.
 
 
-.. note:: Please ensure that the region of your bucket and where you intend to be running the
-          Lightly Docker instance are be the same.
-          E.g. have the instance running in `eu-central-1` and the bucket also in `eu-central-1`.
-          If the region is not the same there can be
-          `degraded transfer speeds and additional costs will be incurred by AWS <https://aws.amazon.com/premiumsupport/knowledge-center/s3-transfer-data-bucket-instance/>`_!
-          This also applies to other cloud providers. We highly recommend using the same region.
+.. note:: 
+  
+  Please ensure that the bucket and the instance running the Lightly Docker are
+  in the same cloud region (S3, Azure) or zone (GCS). E.g. if you are using S3, 
+  have the instance running in `eu-central-1` and the bucket also in 
```

Comment:
E.g. if you are using S3,  have the instance running in `eu-central-1` and the bucket also in 
->
If you are using S3 and your bucket is in `eu-central-1` ensure your EC2 instance is also running in `eu-central-1`